src/io/jesi/backpack/string.cljc

```diff
@@ -134,3 +134,13 @@
   (some-> s
           (str/replace \- \space)
           (->proper-case)))
+
+(defn split-at-first
+  "Splits s at the first occurrence of value"
+  [value s]
+  (when (and (not-blank? value) (not-blank? s))
+    (when-let [idx (str/index-of s value)]
+      (let [val-length (count value)
+            before-split (subs s 0 idx)
+            after-split (subs s (+ idx val-length))]
+        [before-split after-split]))))
```

Comment:
I believe if the `idx` is not found, it should return `[s]` as a result. Similar to other clojure split functions.

https://clojuredocs.org/clojure.core/split-at
https://clojuredocs.org/clojure.string/split

Reply:
Made the modifications to return results to what I think makes sense. See new test cases